pages/common/git-ignore-io.md

```diff
@@ -0,0 +1,12 @@
+# git ignore-io
```

Comment:
Is there any issue with the "command-subcommand-subcommand" pattern? In fact, `git-ignore` is a command on its own (it does not refer to the `git ignore io` command).

Reply:
Yeah, we have issues with this.  The following `tldr` invocations would all display the page `git-ignore-io.md`:
```
tldr git ignore io
tldr git-ignore-io
tldr git-ignore io
tldr git ignore-io
```
The title of the page should be however the command would be invoked on the actual command-line.